pkg/apiserver/index.go

```diff
@@ -25,8 +25,17 @@ import (
 	"github.com/emicklei/go-restful"
 )
 
+const (
+	dashboardAlias = "/ui"
+	dashboardPath  = "/static/app/#/dashboard/"
+)
+
 func IndexHandler(container *restful.Container, muxHelper *MuxHelper) func(http.ResponseWriter, *http.Request) {
 	return func(w http.ResponseWriter, r *http.Request) {
+		if r.URL.Path == "/index.html" || r.URL.Path == dashboardAlias || r.URL.Path == dashboardAlias+"/" {
+			http.Redirect(w, r, dashboardPath, http.StatusTemporaryRedirect)
+		}
+
 		status := http.StatusOK
 		if r.URL.Path != "/" && r.URL.Path != "/index.html" {
```

Comment:
Probably we should remove index.html here if we're going to redirect it to the UI. Clients should specify `/` explicitly, anyway, if they want the resource list.